test/src/test/java/org/corfudb/integration/AbstractIT.java

```diff
@@ -330,16 +330,12 @@ private static List<Long> getChildPIDs(long pid) {
 
     public static long getPid(Process p) {
         long pid = -1;
-
         try {
-            if (p.getClass().getName().equals("java.lang.UNIXProcess") || p.getClass().getName().equals("java.lang.ProcessImpl")) {
-                Field f = p.getClass().getDeclaredField("pid");
-                f.setAccessible(true);
-                pid = f.getLong(p);
-                f.setAccessible(false);
-            }
-        } catch (Exception e) {
-            pid = -1;
+            if (p.getClass().getName().equals("java.lang.UNIXProcess")
+                    || p.getClass().getName().equals("java.lang.ProcessImpl")) {
+                pid = p.pid();
+            }
+        } catch (Exception ignored) {
```

Comment:
Why to ignore the exception? Should we print the exception or log as part of exception?